libnavigation-ui/api/current.txt

```diff
@@ -288,35 +293,71 @@ package com.mapbox.navigation.ui.camera {
 
 package com.mapbox.navigation.ui.feedback {
 
+  public final class FeedbackArrivalFragment extends androidx.fragment.app.DialogFragment {
+    ctor public FeedbackArrivalFragment();
+    field public static final com.mapbox.navigation.ui.feedback.FeedbackArrivalFragment.Companion! Companion;
+  }
+
+  public static final class FeedbackArrivalFragment.Companion {
+    method public com.mapbox.navigation.ui.feedback.FeedbackArrivalFragment newInstance(com.mapbox.navigation.ui.feedback.FeedbackFlowListener feedbackFlowListener);
+  }
+
   public class FeedbackBottomSheet extends com.google.android.material.bottomsheet.BottomSheetDialogFragment implements android.animation.Animator.AnimatorListener {
     ctor public FeedbackBottomSheet();
     method public static com.mapbox.navigation.ui.feedback.FeedbackBottomSheet newInstance(com.mapbox.navigation.ui.feedback.FeedbackBottomSheetListener!, long);
-    method public static com.mapbox.navigation.ui.feedback.FeedbackBottomSheet newInstance(com.mapbox.navigation.ui.feedback.FeedbackBottomSheetListener!, @com.mapbox.navigation.ui.feedback.FeedbackBottomSheet.FeedbackFlowType int, long);
```

Comment:
We deleted `@NonNull` annotation (see comment above). We should put it back to not break SemVer.